app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,6 @@
 class SessionsController < ApplicationController
   # mentioned in lecture but I'm not sure how it applies yet
-  # skip_before_filter :require_login
+  before_action :require_login, only: [:register, :new_seller]
```

Comment:
```suggestion
  before_action :require_login, only: [:register, :new_seller]
  before_action :require_not_seller, only: [:register, :new_seller]

```
This is necessary with the first suggestion